src/lib/core/ripple/ripple-renderer.ts

```diff
@@ -170,25 +169,27 @@ export class RippleRenderer {
 
   /** Sets the trigger element and registers the mouse events. */
   setTriggerElement(element: HTMLElement | null) {
+    const eventOptions = supportsPassiveEventListeners() ? ({passive: true} as any) : false;
```

Comment:
Wouldn't it be enough to check that at `RippleRender` instantiation (e.g. constructor, or property) ?

Reply:
I suppose so, but the check isn't particularly expensive since we cache the result.

Reply:
I'd prefer having it as a property since it's not necessary to have this in the `setTriggerElement` method.

Reply:
Done.